Firestore/core/src/firebase/firestore/model/CMakeLists.txt

```diff
@@ -34,8 +34,6 @@ cc_library(
     resource_path.h
     snapshot_version.cc
     snapshot_version.h
-    timestamp.cc
-    timestamp.h
```

Comment:
Interestingly, I noticed that the other public type, `GeoPoint`, doesn't have it's header in any of the CMake lists. Is that correct?

Reply:
Might be a bug. The CMake rule used to not contain `.h` file and run fine. At certain time, we decided to include the header file in the rule but may missed header during that change.

Reply:
CLion doesn't handle ExternalProjects correctly, so you have to open the Firestore subproject (and tell it where all the rest of externals are). You can do this:
  * Assuming you use `Debug` as your cmake build directory (i.e. `mkdir Debug; cd Debug; cmake ..` to get started)
  * In preferences set CMake Options: `-DCMAKE_INSTALL_PREFIX=/Users/mcg/src/google/ios-sdk/Debug`
  * Generation Path: `/Users/mcg/src/google/ios-sdk/Debug/Firestore`